eq-author-api/schema/resolvers/base.js

```diff
@@ -514,6 +514,8 @@ const Resolvers = {
       }
       return "Read";
     },
+    totalErrorCount: (questionnaire, args, ctx) =>
```

Comment:
Can we add a test on the server api for this.

Reply:
doesnt `totalErrorCount: expect.any(Number),` in `eq-author-api/schema/tests/questionnaire.test.js` test this properly?

Reply:
Good point I missed that.